index.html

```diff
@@ -118,16 +205,62 @@
     // TRANSFORM
     txData = data.slice();
 
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
+    if (options.SOCGroup !== 'all') {
+      var SOCGroup=options.SOCGroup;
       txData = txData.filter(function (d) {
-        return d.year >= yearRange[0] && d.year <= yearRange[1];
+        return d.SuperGroup === SOCGroup;
       });
     }
 
+
     // SCATTERPLOT
 
+    var points=svg.selectAll('.point')
+      .data(txData)
+
+      points.enter().append('circle')
+      .attr('class', 'point')
+      .attr('r', 5)
+
+      points
+      .attr('cx', function(d) { return x(d.WF_Female); })
+      .attr('cy',  function(d) { return y(d.MedEarn); })
+
+
+      points.exit().remove();
+
+
     // HISTOGRAMS
+    var map = data.map(function(d){return d.WF_Female;})
+
+    var histogram = d3.layout.histogram()
+                      .bins(x.ticks())(map);
+
+    var bars = svg.selectAll(".bar")
+                  .data(histogram)
+                  .enter();
+
+   console.log(bars)
+    console.log(bars[0])
+
+     //console.log(d3.extent(bars, function(d) {return d[bars.length]}))
+
+    var yHist = d3.scale.linear()
+      .domain([0,d3.max(bars, function(d) {return d.length})])
```

Comment:
This looks good to me. Curious why you're not using it? Of course, this would also have the bars run the full height of the chart. You might consider adjusting the range to not use the full height.